cmd/trayscale/app.go

```diff
@@ -222,6 +225,7 @@ func (a *App) update(s tsutil.Status) {
 		a.online = online
 		a.notify(online) // TODO: Notify on startup if not connected?
 		systray.SetIcon(statusIcon(online))
+		a.connectionStatusMenuItem.SetTitle(connectionStatusText(online))
```

Comment:
If the user has the tray icon disabled, `a.connectionStatusMenuItem` could be `nil` at this point.

Reply:
Wow, didn't cross my mind to disable tray icon for... TRAYscale 😄 I will work on ensuring we do no-ops for unitialised items sooner then 😉